packages/react-native/ReactCommon/react/renderer/components/view/YogaLayoutableShadowNode.cpp

```diff
@@ -149,12 +139,33 @@ YogaLayoutableShadowNode::YogaLayoutableShadowNode(
         static_cast<const YogaLayoutableShadowNode&>(sourceShadowNode)
             .yogaTreeHasBeenConfigured_;
   }
+                        
+  bool wasDisplayContents = hasDisplayContentsStyle();
```

Comment:
Can we derive this from the source shadow node, instead of the temp state of the Yoga node?

Reply:
It's obvious now that you've pointed it out, done in https://github.com/facebook/react-native/pull/46584/commits/bc1984f5c3ddc2a529c842377a2dbecb7c679dab